### PR TITLE
Complete upgrade to GNOME runtime 3.36

### PR DESCRIPTION
**Pull request by alexander255**
Builds upon #14.

### CODE REVIEW

File: net.poedit.Poedit.json [hunk 1]
```diff
@@ -147,6 +149,14 @@
                     "path": "appdata.patch"
                 }
             ]
+        },
+        {
+            "name": "poedit-fixup",
+            "buildsystem": "simple",
+            "build-commands": [
+                /* Export PoEdit URI handler under allowed name */
+                "mv /app/share/applications/net.poedit.PoeditURI.desktop /app/share/applications/net.poedit.Poedit.URI.desktop"
```
Comment (bilelmoussaoui) [inline net.poedit.Poedit.json:158]: You can do that in `post-install` of the `poedit` module.

**Comment by bilelmoussaoui** [comment]
Thanks a lot :)

File: net.poedit.Poedit.json (hunk 1 above)
Comment (alexander255) [reply net.poedit.Poedit.json:178]: Done!

**Comment by alexander255** [comment]
Thanks! I also added CLD2 to the build to silence the PoEdit warning about language auto-detection. :slightly_smiling_face:

**Comment by bilelmoussaoui** [comment]
Thanks!

**Comment by cho2** [comment]
Thanks @alexander255

**Pull request merged**